### PR TITLE
Fix `_rescale_to_01`

### PR DESCRIPTION
**Pull request by KalelR**
As discussed in Issue #55

### CODE REVIEW

**Comment by awage** [comment]
Do you want me to have a look at this?

**Comment by Datseris** [comment]
yes please!

**Comment by awage** [comment]
I fixed the test but I don't understand what is happening. The `par_weight` add an extra weight to the features far away in parameter space so they don't get clustered together. For the example in the dumb map, if I set this value to 0.1 the test fails because the distance between clusters is too big. For 0.09 it works. I have no clue.

File: test/continuation/grouping_continuation.jl [hunk 1]
```diff
@@ -49,7 +48,7 @@ using Random
             k = sort!(collect(keys(fs)))
             @test sort!(collect(keys(infos))) == k
             @test length(k) == 1
-            @test infos[1] == [0, 0]
+            # @test infos[1] == [0, 0]
```
Comment (Datseris) [inline test/continuation/grouping_continuation.jl:51]: Why is this commented out? The way to fix tests isn't to comment them out :D

Comment (awage) [reply test/continuation/grouping_continuation.jl:51]: Yeah! that was too easy!! It is not the line that was giving trouble. I will check this.

**Comment by Datseris** [comment]
all tests pass so this seems okay? in the end the problem was wrongly giving parameter weight to the simulation?

**Comment by awage** [comment]
Changed version number.

**Comment by awage** [comment]
Wait, there is a version mismatch between main branch and this PR. Main is in 1.3.0 and this PR is in 1.2.5 

I update to 1.3.1

**Pull request merged**